recipes/q/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set version = "1.3.1" %}
 
 package:
-  name: samtools
+  name: Q
```

Comment:
@visze this should be small letters. But it would also be great if the name could be longer. We only have a namespace of 26 for single-letter tools ;)

Reply:
it is not my program. I just want to use it.. I can give him any name but no idea if the developer will like it